src/integrations/elixir.html

```diff
@@ -13,32 +13,44 @@
       integration_feature
         feature="Elixir Code Editor"
         description="Livebook's Elixir code editor supports autocompletion, inline documentation, code formatting, and more."
-        image_src="/images/integration_details/elixir_details_01.png"
+        image_src="/images/integration_details/elixir_code_editor.png"
     }}
     {{>
       integration_feature
         feature="Visual Representation of the running system"
         description="Livebook can render visual representations of supervision trees and message-passing between Elixir processes. <a href='https://hexdocs.pm/kino/Kino.Process.html#seq_trace/2' class='underline'>Learn how to do this.</a>"
-        image_src="/images/integration_details/elixir_details_02.png"
+        image_src="/images/integration_details/elixir_visual_representation_running_system.png"
     }}
     {{>
       integration_feature
         feature="Interactive UI to visualize, debug and edit Elixir pipelines"
         description="Livebook has an interactive widget that lets you see the result of an Elixir pipeline, enable/disable pipeline steps, and re-order them through drag and drop. <a href='https://github.com/livebook-dev/livebook/blob/main/lib/livebook/notebook/learn/intro_to_kino.livemd#dbg' class='underline'>Learn how to do this.</a>"
-        image_src="/images/integration_details/elixir_details_03.png"
+        image_src="/images/integration_details/elixir_interactive_UI.png"
+    }}
+    {{>
+      integration_feature
+        feature="Run a notebook within the context of a Mix project"
+        description="It's super easy to <a href='https://github.com/livebook-dev/livebook/blob/main/lib/livebook/notebook/learn/elixir_and_livebook.livemd#mix-projects' class='underline'>run code of a Mix project inside Livebook</a>. This can be useful for interactive development (like <code>iex -S mix</code>) or for experimenting with your project's code."
```

Comment:
```suggestion
        description="It's super easy to <a href='https://github.com/livebook-dev/livebook/blob/main/lib/livebook/notebook/learn/elixir_and_livebook.livemd#mix-projects' class='underline'>run code of a Mix project inside Livebook</a>. This can be useful for interactive development or for experimenting with your project's code."
```

The link to the source is fine for now, but can you please add to your TODOs to write an article/video on it? Thank you! <3

Reply:
@josevalim 

> but can you please add to your TODOs to write an article/video on it? Thank you! <3

Yes. 👍🙂